enhancements/olm/report-sdk-operators.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Enable tracking of SDK operators in OLM metrics
+authors:
+  - "@varshaprasad96"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-05-28
+status: implementable
+---
+
+# Enable tracking of SDK operators in OLM metrics
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement proposes the addition of operator specific labels to existing OLM metrics, in order to collect data on operators which utilize SDK to build image bundle.
+
+## Motivation
+
+Collecting metrics regarding the usage of Operator SDK among operator authors is important to measure the reachability of the tool. Since OLM already has a pipeline built to obtain metrics on operators and report it to the RedHat telemetry, we intend to utilize the same and obtain data on operators using SDK to build bundle images.
+
+## Goals
+The goal of this enhancement is to list the stamps present in SDK which can be added as labels to OLM metrics.
+
+## Proposal
+
+Currently, there is a [proposal][sdk_metrics_proposal] in review which lists the bundle resources in which stamps would be added indicating the use of SDK to build an operator bundle image. 
+
+The proposed stamps have the following format in `bundle.dockerfile` and `annotations.yaml`:
+1. operators.operatorframework.io.metrics.mediatype.v1 = “metrics+v1”
+2. operators.operatorframework.io.metrics.builder = “operator-sdk-sdkversion”
+3. operators.operatorframework.io.metrics.project_layout = “sdk_project_layout/layout_version” 
+
```

Comment:
So these would be added to every operator that uses the SDK?  How does that fit with the guideline I've heard that oeprators should send back no more than ~10 metrics, does this count for three of those?  Is that still a valid limit?

Reply:
Yes, all operators generated through SDK would have these by default. Also, any other operator owner (who do not use SDK) can use these labels and add their custom values. These are only additional labels to existing metrics sent by OLM (not new metrics as such) as of now.
@ecordell could you please confirm whether these would violate any limit set by OLM/telemetry for its metrics ?

Reply:
Based on the discussions offline, we think the time series limit would not affect us so hard. so having them on operators should be fine.

Reply:
Can you recap the discussion in the proposal as well, I am interested in the cardinality limits as well :) Thanks!